google/cloud/testing_util/opentelemetry_matchers.h

```diff
@@ -198,8 +212,7 @@ ::testing::Matcher<SpanDataPtr> SpanEventsAre(Args const&... matchers) {
  * 1. a new exporter is installed for each test
  * 2. the tests within a fixture do not execute in parallel
  */
-std::shared_ptr<opentelemetry::exporter::memory::InMemorySpanData>
-InstallSpanCatcher();
+std::shared_ptr<SpanCatcher> InstallSpanCatcher();
```

Comment:
Do the comments on this function need to be revisited?

Reply:
Done.

Reply:
There is also mention of `InMemorySpanData::GetSpans()`, which doesn't appear to apply any longer.

Reply:
Argh, missed that. Thanks.